### DISCUSSION TITLE
Chart control in material design

### DISCUSSION THREAD

**Discussion by BabuLalSuthr** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3443] [Q&A] [answered]
Hi Team,

Is there any update on chart control?
Please provide at least simple line chart control as per materialDesign.

**Comment by Keboo** [comment, marked as answer]
At present there are not plans to create a chart control. Though some Material Design libraries include similar, it is not actually something that exists in the [listed components of the spec](https://m3.material.io/components).
